File: src/main/java/no/nav/fo/veilarbregistrering/enhet/adapter/ForretningsAdresseDto.java
package no.nav.fo.veilarbregistrering.enhet.adapter;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class ForretningsAdresseDto {

    private String kommunenummer;
    private GyldighetsperiodeDto gyldighetsperiode;
}

File: src/main/java/no/nav/fo/veilarbregistrering/enhet/adapter/ForretningsAdresseDto.kt
package no.nav.fo.veilarbregistrering.enhet.adapter;

public class ForretningsAdresseDto {

    private final String kommunenummer;
    private final GyldighetsperiodeDto gyldighetsperiode;

    public ForretningsAdresseDto(String kommunenummer, GyldighetsperiodeDto gyldighetsperiode) {
        this.kommunenummer = kommunenummer;
        this.gyldighetsperiode = gyldighetsperiode;
    }

    public String getKommunenummer() {
        return kommunenummer;
    }

    public GyldighetsperiodeDto getGyldighetsperiode() {
        return gyldighetsperiode;
    }
}